src/languages/mojolicious.js

```diff
@@ -0,0 +1,28 @@
+/*
+Language: Perl (Mojolicious)
+Requires: perl.js, xml.js, mojo-ep.js
+Author: Dotan Dimet <dotan@corky.net>
+Description: Perl code with Mojolicious .ep templates embedded in __DATA__ section
+Category: common
+*/
+
+function(hljs) {
```

Comment:
Actually, such "bridge" definitions are better. We had the problem with ERB and VBScript within HTML. Both use the same set of delimiters but VBScript was hard-coded as part of HTML and there was simply no way of having ERB too. Now we have erb.js and vbscript-html.js and everything's fine.